packages/@sanity/components/src/portal/Sticky.js

```diff
@@ -324,6 +316,8 @@ export default class Sticky extends React.PureComponent {
     const {
       useOverlay,
       children,
+      onClickOutside,
+      onEscape,
```

Comment:
`onClickOutside` and `onEscape` are missing in propTypes

Reply:
I think there was i merge-error last night. Sticky should not have any logic of handling escape and click.

Reply:
Ah, right, cause now the <Stacked that we removed yesterday is back. (I thought it was intentional). Maybe you branched off the wrong base? Can probably just reset this as it is in `next` then.